lib/GetStream/Stream/Client.php

```diff
@@ -152,7 +152,7 @@ public function getBaseUrl()
             if ($this->location) {
                 $subdomain = "{$this->location}-api";
                 if ($this->location == 'qa') {
-                    $api_endpoint = 'getstream.io/api';
+                    $api_endpoint = 'stream-io-api.com/api';
                     $this->setProtocol('http');
```

Comment:
please remove this, we now have HTTPS on QA too

Reply:
Will do.